Add a tab for a volume map beside top issue map

dektar opened the pull request:
Shows unscaled / total volumes per state in a week.

nickoneill commented on static/js/main.js:
```diff
@@ -87,25 +106,14 @@ app.model({
-        Raven.captureMessage("Couldn't parse ipinfo json", { level: 'error' });
-      }
+    receiveIPInfoLoc: (state, data) => {
+      geo = response.loc
```
Where is `response` coming from? It works but I do not understand why!

nickoneill commented:
Just that one comment that I want to understand, looks awesome and handles lots of different weird cases!

capndesign commented:
@nickoneill I think this was a magical fat arrow that saved us! I changed `response` to `data`, so we should be good.

nickoneill commented:
Awesome change! This looks great on production

dektar committed:
WIP total calls in state map

dektar committed:
Merge branch 'main' into volumeMap

dektar committed:
Merge branch 'main' into volumeMap

dektar committed:
Allow toggling between total and top calls.

Still needs some a11y markup changes, refactoring for sharing code & cleanliness.

dektar committed:
rename calls -> issues

dektar committed:
Merge branch 'main' into volumeMap

dektar committed:
Key for total calls

dektar committed:
todos

dektar committed:
Merge branch 'main' into volumeMap

dektar committed:
Better colors and don't re-use ol for svg

dektar committed:
Fix label positioning; resolve TODOs; a11y

dektar committed:
format

dektar committed:
Map section title

nickoneill reviewed (approved):
lgtm
